lib/foreman_tasks/dynflow/persistence.rb

```diff
@@ -11,7 +11,8 @@ def save_execution_plan(execution_plan_id, value)
         begin
           on_execution_plan_save(execution_plan_id, value)
```

Comment:
I was thinking in this direction too, but from a different angle - I would like to customize the parameters that are printed for each task - the parameters could be too long, or contain sensitive information.
@iNecas, maybe we need a `#humanize` method in the dynflow task.